docs/concepts/manage-runs.md

```diff
@@ -1,4 +1,102 @@
 (manage-runs)=
 # {octicon}`terminal` Managing Runs
 
-Explain all the function of the run object (logs, status, etc.)
+After you have started a fine-tuning or a synthesis job by calling {meth}`~finetuner.fit` or {meth}`~finetuner.synthesize`,
+you get a {class}`~finetuner.run.Run` object in response.
+This run object serves as a handle for the cloud job that you triggered.
+If you lost the run object for the job, you can create a new `Run` object with the {meth}`finetuner.get_run` function:
+
+```python
+import finetuner
+
+finetuner.login()
+
+run = finetuner.get_run('your-run-name', 'your-experiment-name')
+```
+Since your runs are only accessible to you, you need to login first.
+If you did not specify an experiment name when calling the `fit` or `synthesize` function, 
+you can omit the experiment name in the `get_run` function as well.
+Further, you can use the {meth}`finetuner.list_runs` function to obtain a list of run objects for all jobs you submitted.
+
+```{admonition} Run-time of the list_runs function
+:class: hint
+If this list is very long, the function call might take a while since it needs to send multiple requests.
+```
+
+
+## Attributes of a run
+
+A `Run` object has the following attributes:
+- `name`: It references a job in the Jina AI Cloud. Therefore, it has to be unique for its experiment. If you don't specify a name yourself, Finetuner will give it a memorable name.
+- `config`: A configuration which is serialized from the attributes, which you passed to the `fit` or `synthesize` function.
+- `artifact_id`: The outcome of your job is stored in an artifact on Jina AI Cloud. This attribute holds its id. You can use this artifact id for instance in the {meth}`~finetuner.get_model` function to retrieve the model for inference as explained in the [Inference Section](inference).
+
+## Check the status of a run
+You can check the status of a `Run` via the {meth}`~finetuner.run.Run.status` method:
+
+```python
+print(f'Run status: {run.status()}')
+```
+
+You'll see something like this in the terminal:
+
+```bash
+Run status: CREATED
+```
+
+During fine-tuning or synthesis, the run status changes from:
+1. CREATED: The {class}`~finetuner.run.Run` has been created and submitted to the job queue.
+2. STARTED: The job is in progress.
+3. FINISHED: The job finished successfully and the model has been sent to Jina AI Cloud.
+4. FAILED: The job failed, please check the logs for more details.
+
+## Retrieve the logs of a run
+
+When the status of your run is STARTED or FINISHED,
+you can retrieve logs from the cloud job with the {meth}`~finetuner.run.Run.logs` method.
+This will return a list of all log messages produced by the job so far.
```

Comment:
```suggestion
you can retrieve the job's logs from Jina AI Cloud with the {meth}`~finetuner.run.Run.logs` method.
This will return a list of all log messages produced by the job so far.
```